src/Spec2-Core/SpPresenter.class.st

```diff
@@ -114,7 +114,7 @@ SpPresenter class >> currentApplication [
 { #category : #specs }
 SpPresenter class >> defaultSpec [
 
-	^ self subclassResponsibility
+	"Override to initialize the layout in a non-dynamic way"
```

Comment:
you need to answer nil to make this correct :)

Reply:
ok, done it